### PR TITLE
14239 - Update for 0 admin contacts

### PR DESCRIPTION
**Pull request by kialj876**
*Issue #:*
https://github.com/bcgov/entity/issues/14239

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/Makefile [hunk 1]
```diff
@@ -55,15 +55,12 @@ install: clean ## Install python virtrual environment
 	test -f venv/bin/activate || python3 -m venv  $(CURRENT_ABS_DIR)/venv ;\
 	. venv/bin/activate ;\
 	pip install --upgrade pip ;\
-	pip install -Ur requirements/prod.txt ;\
```
Comment (seeker25) [inline auth-api/Makefile:58]: Looks good

File: auth-api/src/auth_api/services/membership.py [hunk 2]
```diff
@@ -245,14 +248,15 @@ def update_membership(self, updated_fields, **kwargs):
...
+        # send mail to the person itself who is getting removed by staff ;if he is admin and has an email on record
         if user_from_context.is_staff() and not is_bcros_user and admin_getting_removed:
-            recipient_email = ContactLinkModel.find_by_user_id(self._model.user.id).contact.email
-            data = {
-                'accountId': self._model.org.id,
-                'recipientEmail': recipient_email
-            }
-            publish_to_mailer(notification_type='adminRemoved', org_id=self._model.org.id, data=data)
+            contact_link = ContactLinkModel.find_by_user_id(self._model.user.id)
+            if contact_link and contact_link.contact.email:
```
Comment (seeker25) [inline auth-api/src/auth_api/services/membership.py:254]: No logging if no email.. but I think it's fine in this case.

**Review by seeker25 — APPROVED** [review]
Any idea what we should do to get contact links back? Have ops reach out? Maybe request it on the frontend?

**Comment by kialj876** [comment]
> Any idea what we should do to get contact links back? Have ops reach out? Maybe request it on the frontend?

That's a good question. This obviously hasn't been happening often, but there do seem to be a few that get added without a contact every month since the end of 2019. I'll bring this up in standup (I like your frontend suggestion) and we can see how much it matters / if we can get something in grooming

**Pull request merged**